lib/WinEventEmitter.js

```diff
@@ -60,13 +60,19 @@ class WinEventEmitter extends EventEmitter {
 
   start () {
     this._id = setTimeout(() => {
-      const providers = `'${this.options.providers.join('\', \'')}'`;
+      let providers = null 
+      let logName = null
+      if (this.options.providers) {
+        providers = `'${this.options.providers.join('\', \'')}'`;
+      }
+      if (this.options.logNames)
+       logName = `'${this.options.logNames.join('\', \'')}'`;
       let startTime = convertPowershellDate(this.options.startTime), endTime = convertPowershellDate(this.options.endTime);
 
       let creds = '';
       let credsOpt = '';
       if (this.options.userName && this.options.password) {
-        creds =  `$User = '${this.options.userName}'; $Pass = ConvertTo-SecureString -String '${this.options.password}' -AsPlainText -Force; $Credential = New-Object –TypeName 'System.Management.Automation.PSCredential' –ArgumentList $User, $Pass; `;
+        creds =  `$User = '${this.options.userName}'; $Pass = ConvertTo-SecureString -String '${this.options.password}' -AsPlainText -Force; $Credential = New-Object â€“TypeName 'System.Management.Automation.PSCredential' â€“ArgumentList $User, $Pass; `;
```

Comment:
Sorry, I assume notepad.exe or some other tool (remote desktop?) produced this. I had no windows-pc and used remote desktop from a Mac.